src/sentry/digests/notifications.py

```diff
@@ -17,6 +17,33 @@
 
 Notification = namedtuple("Notification", "event rules")
 
+TARGETED_MAIL_ACTION_SYMBOL = "targeted_mail"
+
+
+def is_targeted_action_key(key):
+    return (
+        len(
+            [
+                token
+                for idx, token in enumerate(key.split(":"))
+                if idx == 0 and token == TARGETED_MAIL_ACTION_SYMBOL
+            ]
+        )
+        == 1
+    )
+
+
+def split_key_for_targeted_action(key):
+    from sentry.rules.actions.notify_email import MailAdapter
+
+    _, _, project_id, target_type, target_identifier_str = key.split(":", 4)
+    return (
+        MailAdapter(),
```

Comment:
It's a little weird that we return this here, we could probably just instantiate it when we need it (unless there's some reason that's difficult)